ComputerInterface/BaseGameInterface.cs

```diff
@@ -309,6 +348,12 @@ public enum ETurnMode
             NONE
         }
 
+        public enum EItemMode
+        {
+            FALSE,
+            TRUE
+        }
```

Comment:
Why is this using an enum and not a boolean? See voice chat code for an example of how boolean settings are already implemented